website/content/blog/2024-12-01-a-year-in-review.md

```diff
@@ -0,0 +1,100 @@
+---
+
+# What a year it's been!
+
+It has been a year since we announced that we received a very generous donation from Re-Logic and the formation of the MonoGame Foundation. A lot has happened.
+
+### So far in the last year we:
+
+* Now have recurring Patreon subscriptions of $788/month  
+* Refreshed the website, with more focus on the creators and their titles \- Thanks Chris (ArisTurtle)  
+* Overhauled the documentation site and increased the content by more than 500%, with even more to come  
+* Released 3.8.2
+
+* #### Appointed to the board
+
+  * President \- Dean Ellis  
+  * Chairman \- Marko Jeremic  
+  * Secretary \- Simon Jackson  
+  * Banking \- Tom Spilman  
+  * Member \- Thomas Altenburger  
+  * Treasurer \- Dominique Louis  
+  * Please read our [Board Meeting notes](https://monogame.net/blog/meeting)  
+* Regular board meetings with full transparency, keeping the community involved in the plans and actions for MonoGame to take it into the next decade.
+
+* #### Completed Bounties
+
+  * [Package up ffmpeg and ffprobe](https://github.com/MonoGame/MonoGame/issues/8241)  
+  * [Upgrade MonoGame to use BasisUniversal for cross-platform Texture Compression](https://github.com/MonoGame/MonoGame/issues/8419)  
+  * [A shared base for public and console repos](https://github.com/MonoGame/MonoGame/issues/8242)  
+  * [Implement the Direct3D 12 / GDK/GDKX backend \#8195](https://github.com/MonoGame/MonoGame/issues/8195)
+
+* #### In Progress Bounties
+
+  * [Switch the console runtime from BRUTE to NativeAOT \#8194](https://github.com/MonoGame/MonoGame/issues/8194)  
+  * [A better 2D onboarding tutorial \#8317](https://github.com/MonoGame/MonoGame/issues/8317)  
+  * [A better 3D onboarding tutorial \#8318](https://github.com/MonoGame/MonoGame/issues/8318)
+
+* #### More bounties to come in the areas of:
+
+  * Graphics  
+    * Optimise and fix DesktopVK  
+      * Why: it is still experimental and there is some unexpected flickers  
+  * Content  
+    * Native Texture Compression and Asset processing shared object  
+      * Why: enhance assets processing performance and reduce dependencies  
+    * Upgrade Shader Compiler to use [https://github.com/microsoft/ShaderConductor](https://github.com/microsoft/ShaderConductor) and [https://github.com/KhronosGroup/SPIRV-Cross](https://github.com/KhronosGroup/SPIRV-Cross)	  
+      * Why: retiring mojoshader to allow future improvements on shaders  
+  * Audio  
+    * FAudio implementation for DesktopVK  
+      * Why: replace OpenAL-Soft which can’t be totally XAudio-accurate  
+  * Tutorials and content  
+    * Advanced 3D concepts (Some ideas I know this isn’t a planning doc)  
+      * Custom Effects   
+      * Culling  
+      * Shadows  
+      * Post Processing  
+      * Scene Management?  
+      * Deferred Rendering  
+      * More starter kits for 2D and 3D  
+      * More shader guidance/samples
+
+* #### Bug Fixes
+
+  * Re-enabled the Unit Tests for PR’s.  
```

Comment:
There are allot more bug fixes than this in the last year? (I hope!)

Reply:
@dellis1972 Added more, but not everything. I tried to pick out "highlights" and only closed PRs since out 3.8.2 release in August. Let me know which ones from the closed PR [list](https://github.com/MonoGame/MonoGame/pulls?page=1&q=is%3Apr+is%3Aclosed) I should add.

Reply:
looks better now